src/pantry.js

```diff
@@ -4,4 +4,4 @@ class Pantry {
   }
 }
 
```

Comment:
Also, did you have methods included in this file or is that something you're working on next?

Reply:
That's my bad, I realized I needed to wait for the User class to be all set since the Pantry relies on user data so much. Perhaps I could have started the methods, I was just nervous.